backends/libpurple/main.cpp

```diff
@@ -1537,16 +1537,17 @@ static void *notify_user_info(PurpleConnection *gc, const char *who, PurpleNotif
 		vcardEntry = (PurpleNotifyUserInfoEntry *)(vcardEntries->data);
 		if (purple_notify_user_info_entry_get_label_wrapped(vcardEntry) && purple_notify_user_info_entry_get_value_wrapped(vcardEntry)){
 			label = purple_notify_user_info_entry_get_label_wrapped(vcardEntry);
-			if (label == "Given Name" || label == "First Name") {
+			std::transform(label.begin(), label.end(), label.begin(), ::tolower);
```

Comment:
Please use `boost::locale::to_lower` (as you can see it in this file around) because we need to be compatible with old compilers

Reply:
Done.